File: Do_it_App/4장/SnowAnimation/SnowAnimation/ViewController.m
//
//  ViewController.m
//  SnowAnimation
//
//  Created by 소영섭 on 2014. 7. 29..
//  Copyright (c) 2014년 소영섭. All rights reserved.
//

#import "ViewController.h"
#import "SnowAniViewController.h"


@implementation ViewController

@synthesize infoButton;
- (void)viewDidLoad {
    
     SnowAniViewController *viewController = [self.storyboard instantiateViewControllerWithIdentifier:@"SnowAniViewController"];

    
    //infoButton 뒤로 RecordViewController.view를 넣는다.
    [self.view insertSubview:viewController.view belowSubview:infoButton];
    [super viewDidLoad];
    
}

/*

-(IBAction)PushBackClick
{
    [self dismissViewControllerAnimated:YES completion:nil];
    
}


-(IBAction)PushSetupClick
{
    
    UIViewController *viewController = [self.storyboard instantiateViewControllerWithIdentifier:@"SnowInfoViewController"];
    
    [self presentViewController:viewController animated:YES completion:nil];
    
}
*/


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
@end

File: ImageSelect/ImageSelect/ViewController.swift
//
//  ViewController.swift
//  ImageSelect
//
//  Created by 이상학 on 2015. 10. 5..
//  Copyright © 2015년 ryan. All rights reserved.
//

import UIKit

class ViewController: UIViewController, UIImagePickerControllerDelegate, UINavigationControllerDelegate {
    @IBOutlet weak var myImage: UIImageView!

    @IBOutlet weak var textField: UITextField!
    
    var string = ""
    
    @IBAction func selectImageTapped(sender: AnyObject) { // 사진 불러오기
        var pickerController = UIImagePickerController()
        pickerController.delegate = self // ViewController 로 대행
        pickerController.sourceType = UIImagePickerControllerSourceType.PhotoLibrary // 사진 가져오는 곳은 사진첩
        self.presentViewController(pickerController, animated: true, completion: nil)
    }

    
    func imagePickerController(picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [String : AnyObject]) {
        myImage.image = info[UIImagePickerControllerOriginalImage] as? UIImage
        myImage.backgroundColor = UIColor.clearColor()
        self.dismissViewControllerAnimated(true, completion: nil)
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func completeButtonTapped(sender: AnyObject) {
    }

}

